discovery/fidelity_api.py

```diff
@@ -24,37 +24,39 @@ def orders(l_args):
 
         url_orders = f"https://eresearch.fidelity.com/eresearch/gotoBL/fidelityTopOrders.jhtml"
         text_soup_url_orders = BeautifulSoup(requests.get(url_orders).text, "lxml")
-        
+
         l_orders = list()
         l_orders_vals = list()
         idx = 0
-        for orders in text_soup_url_orders.findAll('td', {'class' : ['second', 'third', 'fourth', 'fifth', 'sixth', 'seventh', 'eight']}):
-            
-            if ((idx+1)%3 == 0) or ((idx+1)%4 == 0) or ((idx+1)%6 == 0):
+        for orders in text_soup_url_orders.findAll('td', {
+            'class': ['second', 'third', 'fourth', 'fifth', 'sixth', 'seventh', 'eight']}):
+
+            if ((idx + 1) % 3 == 0) or ((idx + 1) % 4 == 0) or ((idx + 1) % 6 == 0):
                 l_orders_vals.append(orders.contents[1])
-            elif ((idx+1)%5 == 0):
+            elif ((idx + 1) % 5 == 0):
                 s_orders = str(orders)
                 l_orders_vals.append(s_orders[s_orders.find('title="') + len('title="'): s_orders.find('"/>')])
             else:
                 l_orders_vals.append(orders.text.strip())
-            
+
             idx += 1
-            
+
             # Add value to dictionary
-            if (idx+1)%8 == 0:
+            if (idx + 1) % 8 == 0:
                 l_orders.append(l_orders_vals)
                 l_orders_vals = list()
                 idx = 0
 
-        df_orders = pd.DataFrame(l_orders, columns = ['Symbol', 'Company', 'Price Change', '# Buy Orders', 
-                                                'Buy / Sell Ratio', '# Sell Orders', 'Latest News'])
+        df_orders = pd.DataFrame(l_orders, columns=['Symbol', 'Company', 'Price Change', '# Buy Orders',
```

Comment:
`columns` are mutable default arguments https://docs.python-guide.org/writing/gotchas/

Reply:
Sure, but out of my refactoring scope for this pull request.